app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<% media = Work.spotlight%>
+<%if media%>
+    <section class = "spotlight">
+        <h2 class="spotlight__header">
+        <span class="spotlight__header--prefix">Media Spotlight</span>
+        <%= link_to media.title, work_path(media.id), class:"spotlight__link-to"%> by <%= media.creator%> 
+        </h2>
+        <p class="spotlight__description"><%=media.votes.where(value:1).count%> upvotes, <%=media.votes.where(value:-1).count%> downvotes - <%= media.description%></p>
+    </section>
+<%end%>
+<hr class="root__hr">
+<section class="top-ten__container">
+    <section class="top-ten__list-container">
+        <h2 class="top-ten__header"> Top Books </h2>
+            <ul class="list-group top-ten__list">
+                <% Work.top_ten("book").each do |book|%>
+                    <li class="list-group-item">
+                    <h4><%= link_to book.title, work_path(book.id)%><small class="top-ten__creator"> by <%= book.creator %></small>
+                    </h4> 
+                    <p><%=media.votes.where(value:1).count%> upvotes, <%=media.votes.where(value:-1).count%> downvotes<p>
```

Comment:
This line of code is causing all of the works on your main page to have the same upvote count.